test/Container/ApplicationFactoryTest.php

```diff
@@ -1071,4 +1071,226 @@ public function testCanSpecifyRouteOptionsViaConfiguration()
         $this->assertInstanceOf(Route::class, $route);
         $this->assertEquals($expected, $route->getOptions());
     }
+
+    /**
+     * @expectedException \Zend\Expressive\Container\Exception\InvalidArgumentException
```

Comment:
... I'm thinking of for example: `'must be in form of an array; received "string"'`.